TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/TowerController.java

```diff
@@ -36,31 +38,31 @@ public TowerController (HardwareMap hardwareMap){
         uBar.setDirection(DcMotor.Direction.FORWARD);
 
         //Setup sensors
-        DigitalChannel highSensor = hardwareMap.get(DigitalChannel.class, "highSensor");
-        DigitalChannel lowSensor = hardwareMap.get(DigitalChannel.class, "lowSensor");
-        highSensor.setMode(DigitalChannel.Mode.INPUT);
-        lowSensor.setMode(DigitalChannel.Mode.INPUT);
+//        DigitalChannel highSensor = hardwareMap.get(DigitalChannel.class, "highSensor");
+//        DigitalChannel lowSensor = hardwareMap.get(DigitalChannel.class, "lowSensor");
+//        highSensor.setMode(DigitalChannel.Mode.INPUT);
+//        lowSensor.setMode(DigitalChannel.Mode.INPUT);
 
         //setup encoder
         uBar.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
     }
 
     public void handleScrew() {
         if(raiseTower){
-            if(highSensor.getState()){
-                screw.setPower(0);
-            }
-            else{
+//            if(highSensor.getState()){
+//                screw.setPower(0);
+//            }
+//            else{
                 screw.setPower(1);
-            }
+//            }
         }
         else{
-            if(lowSensor.getState()){
-                screw.setPower(0);
-            }
-            else{
+//            if(lowSensor.getState()){
+//                screw.setPower(0);
+//            }
+//            else{
                 screw.setPower(-1);
-            }
+//            }
         }
     }
 
```

Comment:
Looks like the tower has to be going up or down. Never stopped.  But that isn't part of this commit.  Should we discard this file's changes?